migrate/v0_15/migrate.go

```diff
@@ -0,0 +1,193 @@
+package v0_15
+
+import (
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/genutil"
+
+	cryptoAmino "github.com/tendermint/tendermint/crypto/encoding/amino"
+	tmtypes "github.com/tendermint/tendermint/types"
+
+	"github.com/kava-labs/kava/app"
+	v0_14committee "github.com/kava-labs/kava/x/committee/legacy/v0_14"
+	"github.com/kava-labs/kava/x/committee/types"
+	v0_15committee "github.com/kava-labs/kava/x/committee/types"
+)
+
+var (
+	GenesisTime = time.Date(2021, 4, 8, 15, 0, 0, 0, time.UTC)
```

Comment:
Add TODO to update this

Reply:
Added in https://github.com/Kava-Labs/kava/pull/909/commits/156e6deb9be9f1aaad34e1ae37f8d614b6092fdd